### PR TITLE
Simple Local CMS, closes #329, closes #303, closes #238

### LINKED ISSUE #238: Allow arguments in i18nResolver
They are allowed but not used. Use logic from `CustomI18nHelper` to implement argument replacement and pluralize.

### LINKED ISSUE #303: Unlink "New category" from the home link

### LINKED ISSUE #329: Enable CMS for home banners

### PR DESCRIPTION
**Pull request by lauraluiz**
@schleichardt feel free to review it

### CODE REVIEW

File: shopping-cart/app/shoppingcart/checkout/confirmation/CheckoutConfirmationController.java [hunk 1]
```diff
@@ -36,10 +35,9 @@ public CheckoutConfirmationController(final ControllerDependency controllerDepen
     public F.Promise<Result> show(final String languageTag) {
         final UserContext userContext = userContext(languageTag);
         final F.Promise<Cart> cartPromise = getOrCreateCart(userContext, session());
-        final Http.Context ctx = ctx();
         return cartPromise.map(cart -> {
             final CheckoutConfirmationPageContent content = new CheckoutConfirmationPageContent(cart, userContext, productDataConfig, i18nResolver(), reverseRouter());
-            final SunrisePageData pageData = pageData(userContext, content, ctx);
+            final SunrisePageData pageData = pageData(userContext, content, ctx());
```
Comment (schleichardt) [inline shopping-cart/app/shoppingcart/checkout/confirmation/CheckoutConfirmationController.java]: if you get an error message `Http.Context is not available from here` the reason might be this kind of changes, especially in Play 2.5.

**Comment by schleichardt** [comment]
I only had a quick look, but it looks okay

File: shopping-cart/app/shoppingcart/checkout/confirmation/CheckoutConfirmationController.java (hunk 1 above)
Comment (lauraluiz) [inline shopping-cart/app/shoppingcart/checkout/confirmation/CheckoutConfirmationController.java]: I'll be careful :)

Comment (lauraluiz) [inline shopping-cart/app/shoppingcart/checkout/confirmation/CheckoutConfirmationController.java]: Btw, it should be fixed with specifying a thread with Http default context, shouldn't it?

Comment (schleichardt) [inline shopping-cart/app/shoppingcart/checkout/confirmation/CheckoutConfirmationController.java]: I like it more to capture the stuff from a controller in a value so you need never to worry about it.

Comment (lauraluiz) [inline shopping-cart/app/shoppingcart/checkout/confirmation/CheckoutConfirmationController.java]: I'm actually modifying everything to follow that guideline. I think I will push the changes and dedicate a PR to follow some common standard everywhere.

**Pull request merged**